openstack/sharedfilesystems/v2/shares/results.go

```diff
@@ -110,3 +110,39 @@ type DeleteResult struct {
 type GetResult struct {
 	commonResult
 }
+
+// GrantAccessRes contains all information associated with an OpenStack share Grant Access
+type GrantAccessRes struct {
```

Comment:
There is nowhere else in Gophercloud where "Res" is used. `GrantAccessRes` is not a suitable name. I suggest browsing Gophercloud and finding precedence for a good name to use here.